### PR TITLE
7 post articles articleid comments

### PR DESCRIPTION
**Pull request by Jessel1994**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -201,3 +201,91 @@ describe('GET /api/articles/:article_id/comments', () => {
...
+            body: "Test Comment"
+        }
+        return request(app)
+        .post('/api/articles/10/comments')
+        .send(commentTest)
+        .expect(400)
+        .then((response) => {
+            expect(response.body.msg).toBe('Bad Request')
+        })
+    })
```
Comment (aloush-dev) [inline __tests__/app.test.js]: This should really be a 404 its a good request but the user cannot be found.

File: controllers/comments.controller.js [hunk 2]
```diff
@@ -10,4 +10,27 @@ exports.fetchCommentsByArticleId = (req, res, next) => {
...
+    
+    const {article_id} = req.params
+    if(!comment.hasOwnProperty('username') && !comment.hasOwnProperty('body')) {
+        return res.status(400).send({msg: 'Body is Malformed'})
+        
+
+    } 
+    if (typeof comment.username !== "string" || typeof comment.body !== "string") {
+        return res.status(400).send({msg: 'Schema Validation Failed'})
+    }
```
Comment (aloush-dev) [inline controllers/comments.controller.js]: This logic is fine.
But really we should have this logic in your models, and not directly invoking res from the controller, using promise.reject instead.

**Review by aloush-dev — APPROVED** [review]
Nice job, good testing.
Please address the comments I have left before merging.

**Review by yalicee — APPROVED** [review]
👍

**Pull request closed without merging**